modules/installation-about-restricted-network.adoc

```diff
@@ -14,13 +20,15 @@ not infrastructure that the installation program provisions, so your platform se
 limited.
 // maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
 
+ifndef::ibm-power[]
```

Comment:
Is this content okay to include in Z or should it also be ifdef'd out?

Reply:
@bmcelvee That's right, this needs to be absent for IBM Power only. It should stay for IBM Z.